code/src/Album.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Artist } from "./Artist";
+import { ReactComponent as PlayButton } from './icons/play.svg';
+import { ReactComponent as HeartButton } from './icons/heart.svg';
+import { ReactComponent as DotButton } from './icons/dots.svg';
+
+//props are passed into components like arguments are into function
+export const Album = (props) => {
+
+  //create array of objects containing artist name, URL for each album.
+  const artistsArr = props.artists.map((artist) => {
+    const name = artist.name;
+    const url = artist.external_urls.spotify;
+    return { name, url };
+  });
+
+
+  return (
+    <>
```

Comment:
If your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```
return (
    <div>My element 1</div>
    <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```
return (
    <>
        <div>My element 1</div>
	<div>My element 2</div>
    </>
);
```

But in this case WE DON'T NEED to use React Fragment :

```
return (
    <div>My element 1</div>
);
```